test/run_clippy.sh

```diff
@@ -8,12 +8,11 @@ set -euo pipefail
 source ./support/ci/shared.sh
 
 # Because sadness
-sudo chown buildkite-agent /home/buildkite-agent
+if [[ "${BUILDKITE:-}" = "true" ]]; then
```

Comment:
This will work, but it can be simplified a bit an account of [`true` being a POSIX command](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/true.html) whose purpose is to exit 0.
```suggestion
if ${BUILDKITE:-false}; then
```

If you prefer it the other way, that's fine too.

Reply:
yeah that is better. thanks!